management/dynamic.py

```diff
@@ -17,7 +17,10 @@ def jset(variable,value):
     Keyword arguments:
     variable -> the variable that should be changed.  
     value -> the value the variable should be set to."""
-
+    
+    jdict1 = json.loads(open(dynamic_config).read())
+    jdict1[variable] = value
```

Comment:
Does this write back to disk?

Reply:
erm.. I might have forgotten that bit
:p

Reply:
I'll be right back